script/ci

```diff
@@ -1,3 +1,7 @@
 #!/usr/bin/env bash
 
-sbt clean compile test riffRaffUpload
+sbt clean compile test assembly
+
+mkdir cfn
+mv cloudformation/anghammarad.template.yaml cfn/cfn.yml
```

Comment:
You can simplify by doing this work in the contentDirectories section of guardian/actions-riff-raff inputs. I.e. remove these two lines and update the action input to:

```
...
cfn:
  - cloudformation/anghammarad.template.yaml
```

Reply:
https://github.com/guardian/anghammarad/pull/87